### PR TITLE
Infinity suggestions for deployment

### LINKED ISSUE #214: infinity suggestions
Cool work! Might feature it on Social Media / Twitter.

I recently added multi-model deployments that default to the ENV variables. Aka you can set the default env variable 
-> `--port` -> `INFINITY_PORT` or `--batch-size` -> `INFINITY_BATCH_SIZE`. Multiple args can be separated with `;`, which works if the cli arg can be overloaded. 

```bash
--model-id ${INFINITY_EMBEDDING_MODEL} --model-id ${INFINITY_RERANKING_MODEL}
```

```bash
INFINITY_MODEL_ID=mixedbread-ai/mxbai-embed-large-v1;mixedbread-ai/mxbai-rerank-xsmall-v1;
```

https://huggingface.co/spaces/michaelfeil/infinity/tree/305b1c2b583e9968aa153c45e8e50555af1d9575

### PR DESCRIPTION
**Pull request by michaelfeil**
Great work, here are some suggestions. 

Closes #214 

Please review, I just "eye-balled' the edits in Github UI without broad testing. Pretty confident that it works, except for typos.

### CODE REVIEW

**Comment by S1LV3RJ1NX** [comment]
@michaelfeil, we are trying to see if everything works fine for our users, it will help if you take the latest pull from `main` and the changes above to see if docker-compose works fine? Will that be possible?

Thanks!

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ docker-compose --env-file compose.env up --build
 -   The compose file uses `compose.env` file for environment variables. You can modify it as per your needs.
 -   The compose file will start the following services:
     -   `ollama-server` - Used to start local LLM server. `compose.env` has `OLLAMA_MODEL` as the environment variable to specify the model.
-    -   `infinity-server` - Used to start local embeddings and rerankers server. `compose.env` has `INFINITY_EMBEDDING_MODEL` and `INFINITY_RERANKING_MODEL` as the environment variable to specify the embedding and reranker.
+    -   `infinity-server` - Used to start local embeddings and rerankers server. `compose.env` has `INFINITY_MODEL_ID` as the environment variable to specify the embedding and reranker.
```
Comment (chiragjn) [inline README.md:91]: Let's revert this and instead format the `INFINITY_MODEL_ID` in the docker compose itself
Easier for the users to manage them differently

File: compose.env [hunk 2]
```diff
@@ -5,9 +5,8 @@ OLLAMA_HOST=0.0.0.0
 
 
 ## INFINITY VARS
-INFINITY_EMBEDDING_MODEL=mixedbread-ai/mxbai-embed-large-v1
-INFINITY_RERANKING_MODEL=mixedbread-ai/mxbai-rerank-xsmall-v1
-INFINITY_BATCH_SIZE=4
+INFINITY_EMBEDDING_MODEL="mixedbread-ai/mxbai-embed-large-v1;mixedbread-ai/mxbai-rerank-xsmall-v1"
```
Comment (chiragjn) [inline compose.env:8]: We will keep them separate for the user to edit

File: docker-compose.yaml [hunk 3]
```diff
@@ -34,7 +35,11 @@ services:
             interval: 1m
             timeout: 10s
             retries: 3
-        command: v2 --model-id ${INFINITY_EMBEDDING_MODEL} --model-id ${INFINITY_RERANKING_MODEL} --batch-size ${INFINITY_BATCH_SIZE}
+            start_period: 60s
+        environment:
+            - INFINITY_MODEL_ID=${INFINITY_MODEL_ID}
```
Comment (chiragjn) [inline docker-compose.yaml:40]:
```suggestion
            - INFINITY_MODEL_ID=${INFINITY_EMBEDDING_MODEL};${INFINITY_RERANKING_MODEL}
```

File: README.md (hunk 1 above)
Comment (michaelfeil) [reply README.md:91]: Are you sure? Feel free to commit on the branch!

File: compose.env (hunk 2 above)
Comment (michaelfeil) [reply compose.env:8]: Okay!

**Review by chiragjn — APPROVED** [review]

**Pull request merged**